src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/mgmt_vmss/azuredeploy.json

```diff
@@ -90,6 +90,13 @@
         "description": "Name of load balancer backend pool."
       }
     },
+    "loadBalancerNatPoolName": {
+      "type": "string",
+      "defaultValue": "[concat(parameters('name'), 'natpool')]",
+      "metadata": {
+        "description": "Name of load balancer NAT pool."
```

Comment:
we're creating a new Nat pool here, so only the name is required

---
In reply to: [76827837](https://github.com/Azure/azure-cli/pull/774#discussion_r76827837) [](ancestors = 76827837)